packages/projection/test/operators/Mappers/withHandle.test.ts

```diff
@@ -0,0 +1,88 @@
+import { Cardano } from '@cardano-sdk/core';
+import { ProjectionEvent } from '../../../src';
+import { filterHandlesByPolicyId, withHandles } from '../../../src/operators/Mappers';
+import { firstValueFrom, of } from 'rxjs';
+
+describe('withHandles', () => {
+  it('maps tx outputs to handles and filter by policyId', async () => {
+    const validTxSource$ = of({
+      block: {
+        body: [
+          {
+            body: {
+              outputs: [
+                {
+                  address: Cardano.PaymentAddress('addr_test1wzlv9cslk9tcj0wpm9p5t6kajyt37ap5sc9rzkaxa9p67ys2ygypv'),
+                  datumHash: '99c170cc1247e7b7971e194c7e400e219360d3991cb588e9833f77ee9edbbd06' as Cardano.DatumHash,
+                  value: {
+                    assets: new Map([
+                      [
+                        Cardano.AssetId(
+                          '8f78a4388b1a3e1a1435257e9356fa0c2cc0d3a5999d63b5886c96435365636f6e6454657374746f6b656e'
```

Comment:
Let's split the `assetId` down into it's components of `policyId` and unencoded `assetName` to expose the handle in readable form, to make the test make more reasonable.